libconsensus/pbft/PBFTEngine.cpp

```diff
@@ -548,6 +548,8 @@ void PBFTEngine::checkSealerList(Block const& block)
 /// check Block sign
 bool PBFTEngine::checkBlock(Block const& block)
 {
+    Guard l(m_mutex);
+    resetConfig();
```

Comment:
for what ? plz add some comments

Reply:
Verifiy block is faster than block report. checkBlock() need to resetConfig to update it's sealer cache.